pysat/_meta.py

```diff
@@ -270,12 +270,18 @@ def drop(self, names):
                 _ = self._ho_data.pop(name)
 
     def keep(self, keep_names):
-        """Keeps variables (keep_names) while dropping other parameters"""
+        """Keeps variables (keep_names) while dropping other parameters
 
+        Parameters
+        ----------
+        keep_names : list-like
+            variables to keep
+        """
+        keep_ = [self.var_case_name(name) for name in keep_names]
         current_names = self._data.index
         drop_names = []
         for name in current_names:
-            if name not in keep_names:
+            if name not in keep_:
```

Comment:
I thought it was convention to only use a single trailing underscore to avoid a name conflict with an existing python variable/function.  That doesn't seem to be the case with `keep`.  Am I missing something?